web/cobrands/fixmystreet/assets.js

```diff
@@ -385,6 +385,11 @@ function check_zoom_message_visibility() {
     if (this.relevant()) {
         if ($p.length === 0) {
             $p = $("<p>").prop("id", id).prop('class', 'category_meta_message');
+            if ($('html').hasClass('mobile')) {
+                $p.click(function() {
+                    $("#mob_ok").trigger('click');
```

Comment:
This is nice; additionally I feel we should probably lose the "Try again" overlay entirely, perhaps? So that people can scroll back up the page rather than have to click either this or the "Map" link that they might not realise is a link.

Reply:
Possibly, but let's consider that in a separate issue: #2667.